From 7be849c9606d44fc4260c817650eaa1828f13c76 Mon Sep 17 00:00:00 2001
From: "Pillai, Venkatesh" <venkatesh.pillai@intel.com>
Date: Thu, 24 Aug 2023 09:57:56 +0530
Subject: [PATCH] Add ICameraProvider aidl hal service contexts

Added external camera provider aidl service to hal camera service
contexts.

Tracked-On: OAM-110079
Signed-off-by: Pillai, Venkatesh <venkatesh.pillai@intel.com>
---
 camera-ext/ext-camera-only/service_contexts | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 camera-ext/ext-camera-only/service_contexts

diff --git a/camera-ext/ext-camera-only/service_contexts b/camera-ext/ext-camera-only/service_contexts
new file mode 100644
index 0000000..631d8c7
--- /dev/null
+++ b/camera-ext/ext-camera-only/service_contexts
@@ -0,0 +1 @@
+android.hardware.camera.provider.ICameraProvider/external/0    u:object_r:hal_camera_service:s0
-- 
2.40.0

